### PR TITLE
Modernize the gem, add travis

### PR DESCRIPTION
**Pull request by tamird**
@emerose Hi! this is currently blocking web from going to 2.0.0

### CODE REVIEW

**Comment by tamird** [comment]
cc @strzalek

File: pbkdf2.gemspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
+$LOAD_PATH.unshift File.expand_path('../lib', __FILE__)
+require 'pbkdf2'
+
+Gem::Specification.new do |s|
+  s.name = 'pbkdf2-ruby'
```
Comment (lukaszx0) [inline pbkdf2.gemspec]: should be just `pbkdf2`

Comment (tamird) [inline pbkdf2.gemspec]: thanks fixed

**Comment by lukaszx0** [comment]
:+1:

**Comment by emerose** [comment]
Merging without testing, because I trust you guys so.

**Comment by tamird** [comment]
@emerose would you mind putting up a new release on rubygems?

**Comment by tamird** [comment]
also you'll need to do a bit of work to add this on https://travis-ci.org

**Pull request merged**